docs/configuration_options.rst

```diff
@@ -98,4 +98,23 @@ Configuration Options
 
   * Type: boolean
   * Default: true
-  * Importance: low
\ No newline at end of file
+  * Importance: low
+
+``table.types``
```

Comment:
This file is actually automatically generated from the `ConfigDef`, so the changes here will get overwritten when someone regenerates this. Do you want to include these in the doc string too so it'll match? (You can generate the file by running the `main()` method in SchemaRegistryConfig)